vaadin-dir-mixin.html

```diff
@@ -0,0 +1,97 @@
+<script>
+(function() {
+  /**
+   * Array of Vaadin custom element classes that have been subscribed to the dir changes.
+   */
+  const directionSubscribers = [];
+  const directionUpdater = function() {
+    if (directionSubscribers) {
```

Comment:
This `if` will always return `true` so you could remove it now.

Reply:
Removed.